src/db/tripledb.plt

```diff
@@ -0,0 +1,60 @@
+:- begin_tests('db/tripledb').
+
+:- use_module(library('db/tripledb'),
+    [ tripledb_load/2,
+      tripledb_ask/3,
+      tripledb_forget/3,
+      tripledb_tell/3
+    ]).
+
+% load swrl owl file for tripledb testing
+test('tripledb_load(URL)') :-
+  tripledb_load(
+    'package://knowrob/owl/test/swrl.owl',
+    [ graph(common),
+      namespace(knowrob, 'http://knowrob.org/kb/swrl_test#')
+    ]).
+
+% check via tripledb_ask if individual triple exists
+test('tripledb_ask(S,P,O)') :-
+  tripledb_ask(
+    knowrob:'Adult',
+ 	rdfs:'subClassOf',
```

Comment:
still the indentation seems wrong here ;)

Reply:
this is strange. I dont see it in my IDEA.

Reply:
check for the use of tab  vs. spaces.
The Prolog code mostly uses two spaces, but actually I would prefer Tab's instead of spaces

Reply:
hmm. I tried to use tabs for all the parameters, still they do not align themselves! It is very strange. I tried to google pycharm code formating for prolog code but there is not much help!

Reply:
well I can see it in github that line 22 has a tab and some spaces, while line 21 has only spaces